resources/hyperv/environment.go

```diff
@@ -0,0 +1,65 @@
+package hyperv
+
+import (
+	config "github.com/DataDog/test-infra-definitions/common/config"
+	"github.com/DataDog/test-infra-definitions/common/namer"
+
+	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
+)
+
+const (
+	hvConfigNamespace = "hyperv"
+	hvNamerNamespace  = "hv"
+
+	// HyperV Infra (local)
+	DDInfraDefaultPublicKeyPath      = "hv/defaultPublicKeyPath"
+	DDInfraDefaultPrivateKeyPath     = "hv/defaultPrivateKeyPath"
+	DDInfraDefaultPrivateKeyPassword = "hv/defaultPrivateKeyPassword"
+)
+
+type Environment struct {
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
Could we rename to `HyperVEnvironment` ? I was confused between resource environment and command environment, which are the env vars.

Reply:
No because the package is `hyperv` already, and as `govet` says, that stutters.
(Note that we use the same pattern everywhere)

Reply:
True, we do have the `aws.Environment`. Could we add an explicit check for the interface implementation ? It helps me with discoverability.